.vscode/settings.json

```diff
@@ -15,5 +15,6 @@
   "eslint.format.enable": true,
   "editor.codeActionsOnSave": {
     "source.fixAll": true
-  }
+  },
```

Comment:
is this required?

Reply:
I think the linter just likes trailing commas 🤷🏻